Tribler/Core/Modules/restapi/multichain_endpoint.py

```diff
@@ -313,7 +314,8 @@ def render_GET(self, request):
 
         nodes, edges = mc_community.get_graph(unhexlify(focus_node), neighbor_level)
 
-        return json.dumps({"focus_node": focus_node,
+        return json.dumps({"user_node": hexlify(mc_community.my_member.public_key),
+                           "focus_node": focus_node,
```

Comment:
I already tried that. It could work but this way is better since the data processor can be state-less (it does not need a memory to remember what the users pk was). Furthermore, this adds value to the response since the 'user' is important in every view